rules.md

```diff
@@ -1,21 +1,85 @@
 ---
 layout: single-page-md
-title: APIO 2020 Rules
+title: APIO 2020 Competition Rules
 key: rules
 ---
 
-### Online Contest Format
-* The APIO 2020 online contest is a 5-hour contest with flexible start time within a determined window.
-* The contest will be held between Friday, 8 May 2020 (UTC+0 06.00) and Sunday, 10 May 2020 (UTC+0 06.00).
-* Each country must run one or more official contest sites.
-* All participants must take the contest at an official contest site.
-* Each contest site starts the contest at the time of its own choice within the window. Different sites from within the same country may also start the contest at different times.
-* Each contest site must be fully supervised in order to guarantee that the contest rules be observed by all the students.
-* Each country may consist of up to 60 students. However, only the top six students will form the official team, according to the contest results.
-* Students are eligible for this contest if and only if they are eligible for IOI 2020.
-
-### Medal Distribution
-The medal awards are uniquely determined by the following rules:
-* The score necessary to achieve a gold medal is the largest score such that at least one twelfth of all official-team contestants receive a gold medal.
-* The score necessary to achieve a silver medal is the largest score such that at least one fourth of all official-team contestants receive a silver or a gold medal.
-* The score necessary to achieve a bronze medal is the smallest score such that at most one half of all official-team contestants receive a medal.
\ No newline at end of file
+These Competition Rules cover General, Problem Scoring, Contestant Scoring, Grading System, and Clarifications in APIO 2020.
+Delegation Leaders have the responsibility of ensuring that all members of their delegation fully understand these rules and abide by them.
+
+<span style="color:red">**Please do not share the problems after the contest within 2 days since the competition is not held within a same time range.**</span>
```

Comment:
```suggestion
<span style="color:red">**Please do not share the problems after the contest within 2 days since the competition is not held within the same time range.**</span>
```

Reply:
Done.